### PR TITLE
Add initial list of projects.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This isn't everything but it's everything I could think of in one go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+* [ROS Answers](https://answers.ros.org)
+* [ROS Build farm repositories](https://repositories.ros.org)
+* [ROS Build farm](https://build.ros.org)
+* [ROS Discourse](https://discourse.ros.org)
+* [ROS bootstrap repositories](https://repos.ros.org)
+* [ROS package repositories](http://packages.ros.org)
+* [ROS snapshot repositories](http://snapshots.ros.org)
+* [ROS static downloads](http://download.ros.org)
+* [ROS web site](https://www.ros.org)
+* [ROS wiki](https://wiki.ros.org)
```
Comment (clalancette) [inline README.md:26]: I think we should probably add:

* [ROS Documentation](https://docs.ros.org)
* [ROS 2 API documentation](https://docs.ros2.org)
* [ROS package index](https://index.ros.org)

Comment (nuclearsandwich) [reply README.md:26]: Make it so

**Review by clalancette — APPROVED** [review]
Looks good to me.

**Review by tfoote — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
 
 This repository's [issue tracker](https://github.com/osrf/infrastructure/issues) is used to receive reports from the Gazebo and ROS community regarding problems with the infrastructure provided by Open Robotics.
 
+Managed projects include
+
+* [Gazebo Answers](https://answers.gazebosim.org)
+* [Gazebo Community](https://community.gazebosim.org)
+* [Gazebo web site](https://gazebosim.org)
```
Comment (j-rivero) [inline README.md:10]:
```suggestion
* [Gazebo Models web site](http://models.gazebosim.org)
* [Gazebo web site](https://gazebosim.org)
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,31 @@
...
+* [ROS Discourse](https://discourse.ros.org)
+* [ROS bootstrap repositories](https://repos.ros.org)
+* [ROS package repositories](http://packages.ros.org)
+* [ROS snapshot repositories](http://snapshots.ros.org)
+* [ROS static downloads](http://download.ros.org)
+* [ROS web site](https://www.ros.org)
+* [ROS wiki](https://wiki.ros.org)
+* [ROS Documentation](https://docs.ros.org)
+* [ROS 2 API documentation](https://docs.ros2.org)
+* [ROS package index](https://index.ros.org)
```
Comment (j-rivero) [inline README.md:30]:
```suggestion
* [ROSCon conference web site](https:/roscon.ros.org)
* [ROS package index](https://index.ros.org)
```

**Review by j-rivero — COMMENTED** [review]
Added Gazebo models website and ROSCon as suggestions.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,31 @@
...
+* [OSRF package repositories](https://packages.osrfoundation.org)
+* [ROS 2 Build farm repositories](https://repo.ros2.org)
+* [ROS 2 Build farm](https://build.ros2.org)
+* [ROS 2 CI](https://ci.ros2.org)
+* [ROS Answers](https://answers.ros.org)
+* [ROS Build farm repositories](https://repositories.ros.org)
+* [ROS Build farm](https://build.ros.org)
+* [ROS Discourse](https://discourse.ros.org)
+* [ROS bootstrap repositories](https://repos.ros.org)
+* [ROS package repositories](http://packages.ros.org)
```
Comment (j-rivero) [inline README.md:22]:
```suggestion
* [ROS mailing lists](http://http://lists.ros.org)
* [ROS package repositories](http://packages.ros.org)
* [ROS planet web site](https://planet.ros.org/)
* [ROS robots web site](https://robots.ros.org/)
```

**Review by j-rivero — COMMENTED** [review]
Found some more sites using a subdomain search engine. There are some that I'm not sure if they fit well here: navigation.org.org and moveit.ros.org.

File: README.md (hunk 4 above)
Comment (tfoote) [reply README.md:22]: I'd skip this one. It's only legacy/archive now, and been that way for many years.

navigation and moveit are on the cusp. We don't really maintain them and their in github pages, but I guess we could have them here and then forward on.

**Comment by tfoote** [comment]
I'm going to land this so we get the update. We can always update again later.

**Pull request merged**